sources/packages/backend/apps/api/src/testHelpers/auth/aest-token-helpers.ts

```diff
@@ -0,0 +1,53 @@
+import { UserPasswordCredential } from "@sims/utilities/config";
+import { AuthorizedParties } from "../../auth";
+import { getCachedToken } from "./token-helpers";
+
+/**
+ * Known AEST groups. The API right now performs authorization for
+ * Ministry users based on the roles associated to these groups.
+ * These groups represents the different logins with the different
+ * set of roles to be used to execute E2E tests when roles validations
+ * are required.
+ */
+export enum AESTGroups {
+  BusinessAdministrators = "business-administrators",
+  Operations = "operations",
+  OperationsAdministrators = "operations-administrators",
+  MOFOperations = "mof-operations",
```

Comment:
Just for my info: what does MOFOperations mean?

Reply:
I have no idea 😄 
FO stands for "Financial Operations" if I am not wrong.
To us, what matters are the roles associated with this group and which expoints they have access to the API 😉